### ISSUE TITLE
Bags not refreshed and icons swapped or missing

### ISSUE THREAD

**Issue by wrufeger** [arkayenro/arkinventory#1508]
**What version number you are using?**
3.09.55

**What game client are you playing? Retail, Burning Crusade, Classic / Live, PTR, Beta**
Retail

**What language is the game client set to?**
german

**Describe the bug**
Since last addon update bags are not refreshed anymore. Getting additional items or selling junk results in missing icons, swapped icons and other weired stuff. Only hard reload fixes this. As this is my favorite Bag addon with tons of custom rules, it would be nice to have this fixed. Thanks and keep up great work.

**Screenshots**
![image](https://user-images.githubusercontent.com/524191/130415982-275bfbf3-494d-4d2e-a41d-a8118d43920a.png)

**Comment by arkayenro** [comment]
try [alpha 12](https://www.curseforge.com/wow/addons/ark-inventory/files/3434087)

**Comment by DannyJo** [comment]
I have the same issue in BCC but I've just tried alpha 12 and so far it looks like this is fixed 👍

**Comment by Laumesis** [comment]
Yeah. v3.09.55 seems **dangerous** to use. High chance to vendoring wrong items :(
BCC here. Going to roll back ro switch to alpha.
